docs/installation.md

```diff
@@ -25,6 +25,16 @@ The Lychee gallery has a few system requirements. You will need to make sure you
    		- Tokenizer
    		- XML
    		- ZIP
+   	- These PHP extensions are necessary if you are running a FreeBSD system:
+   	 	- Simplexml
+		- Dom
+   	 	- Session
+   	 	- Zlib
+   	- For database, you will need these PHP extensions:
+   	 	- SQLite3 (For Lychee's inbuilt SQLite3)
+   	 	- MySQLi (For MySQL or MariaDB)
+   	 	- PDO_MySQL (For MySQL or MariaDB)
```

Comment:
Wording, and added pgsql as it wasn't mentioned. @nagmat84 Does this wording satisfy your preferred vs legacy concern, or would you prefer it to be more explicit?

```suggestion
   	- You will also need one of these PHP extensions:
   	 	- SQLite3 for SQLite3 databases
   	 	- MySQLi (or PDO_MySQL) for MySQL or MariaDB databases
   	 	- PgSql (or PDO_PGSQL) for PostgreSQL databases
```